config/crossSections.yaml

```diff
@@ -160,39 +160,39 @@ ZZTo4L:
   unc: 0.00122
 ZZTo2L2Nu:
   crossSec: 0.9738
-  reference: XSDB ZZTo2L2Nu_TuneCP5_13TeV_powheg_pythia8 modifiedOn 2021-09-28 13:26:27 mll>4GeV
+  reference: XSDB ZZTo2L2Nu_TuneCP5_13TeV_powheg_pythia8 modifiedOn 2021-09-28 13:26:27
   unc: 0.0009971
 ZZTo2Q2L:
-  crossSec:  3.693e # 5.52
-  reference: ttH AN-19-111
-  unc: 4.149e-03
+  crossSec:  2.214
+  reference: XSDB ZZTo2Q2L_14TeV_powheg_pythia8 modifiedOn 2019-11-11 18:49:02 mll>4GeV
+  unc: 0.001657
 ZZTo2Q2Nu:
   crossSec: 4.561
   reference: XSDB ZZTo2Q2Nu_TuneCP5_13TeV modifiedOn 2020-09-25 05:03:18
   unc: 0.01533
 ZZTo4Q:
   crossSec:  3.302 #6.912 <-- previous value. The new one has been evaluated on many MiniAOD
-  reference: unknown
+  reference: from GenXSecAnalyzer
```

Comment:
when values is computed using GenXSecAnalyzer, specify the dataset name that was used

Reply:
ok